webpack-widget.js

```diff
@@ -91,6 +85,141 @@ const getLegacyWidgetBuildConfig = (config = {}) => {
 			toType: 'dir'
 		}
 	]
+}	
+
+const getDefaultJSRules = () => {
```

Comment:
this guy's name might be a little misleading, it contains css/sass rules too

Reply:
Agree. "defaultRules" might be more fitting.